c2corg_ui/templates/article/index.html

```diff
@@ -0,0 +1,39 @@
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="../helpers/list.html" import="add_doctype_selector"/>
+
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Articles')">${show_title('Articles')}</title></%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection', null);
+  module.value('advancedSearchFilters', {
+    'act': {'type': 'list'},
+    'cat': {'type': 'list'},
+    'atyp': {'type': 'list'},
+    'l': {'type': 'list'}
+  });
+</%block>
+
+<header class="page-main-title">
+  <h3><span translate>Articles</span>
+    <span class="badge" ng-cloak>{{resCounter}}</span>
+  </h3>
+</header>
+<section class="documents-list-section">
+  <%include file="filters.html" />
+  <div class="documents-nav-buttons text-center">
+    <button class="btn btn-default show-documents-filters-phone" translate>Filters</button>
+    ${add_doctype_selector()}
```

Comment:
We don't need the doctype selector in the article advanced search, since it's not part of the guidebook.